doc/examples/sample-ml-app/sample-ml-app/sample-ml-app.vcxproj

```diff
@@ -104,15 +104,15 @@
       <SDLCheck>true</SDLCheck>
       <PreprocessorDefinitions>_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <ConformanceMode>false</ConformanceMode>
-      <AdditionalIncludeDirectories>C:\mlpack\armadillo-8.500.1\include;C:\mlpack\mlpack-3.4.2\build\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>C:\mlpack\armadillo-8.500.1\include;C:\mlpack\mlpack-4.0.0\build\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
Haven't tested the example application build yet, can try this later today, unless someone already tested it?